lib/browser/browser.ts

```diff
@@ -311,7 +312,21 @@ class OktaAuthBrowser extends OktaAuthBase implements OktaAuth, SignoutAPI {
     return this.token.revoke(accessToken);
   }
 
-  // Revokes accessToken, clears all local tokens, then redirects to Okta to end the SSO session.
+  // Revokes the refresh token for the application session
+  async revokeRefreshToken(refreshToken?: RefreshToken) {
+    if (!refreshToken) {
+      refreshToken = (await this.tokenManager.getTokens()).refreshToken as RefreshToken;
```

Comment:
It might be better to get the token from authState (sync access), but as we are also using the same logic in `revokeAccessToken`, the change can be made in a separate PR.

Reply:
We get into a circular dependency if we try that.   I brought it up and Aaron suggested we just do this.

Reply:
wondering how the circular dep happens? it should just be an access to the in-memory state

```
const { refreshToken } = this.authStateManager.getAuthState();
```

Reply:
I'm going off memory, but as I recall `this.authStateManager` didn't exist here.